src/Microsoft.AspNetCore.Mvc.Razor.Extensions/ModelDirective.cs

```diff
@@ -0,0 +1,117 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.AspNetCore.Razor.Evolution;
+using Microsoft.AspNetCore.Razor.Evolution.Intermediate;
+
+namespace Microsoft.AspNetCore.Mvc.Razor.Extensions
+{
+    public static class ModelDirective
```

Comment:
Shouldn't act on this now but thought I'd bring it up. @rynowak / all what do you think about making `@model` part of core Razor?